packages/logger/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
     "extends": "../../tsconfig.base.json",
+    "lib": ["es6", "dom"],
+    "types": ["reflect-metadata"],
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
```

Comment:
shouldnt this be in base?

Reply:
moving to base breaks repo build. need more investigation.